syntax = "proto3";
import "proto/transmission_object.proto";

option go_package = ".;echo";
option java_package = "echo";
package echo;

message EchoRequest {
    echo.TransmissionObject from_client = 1;
}

message EchoResponse {
    echo.TransmissionObject from_server = 1;
}

service Transceiver {
    rpc Echo (EchoRequest) returns (EchoResponse);
}